bulbs/liveblog/views.py

```diff
@@ -1,23 +1,26 @@
 from django.conf import settings
 from django.db.models.loading import get_model
-from django.shortcuts import get_object_or_404
-from django.utils import dateparse
+from django.utils import timezone
 from django.views.decorators.cache import cache_control
-from django.views.generic import TemplateView
+
+from bulbs.content.views import BaseContentDetailView
 
 LiveBlogModel = get_model(settings.BULBS_LIVEBLOG_MODEL)
 
 
-class LiveblogNewEntriesView(TemplateView):
+class LiveblogNewEntriesView(BaseContentDetailView):
+    model = LiveBlogModel
     template_name = 'liveblog/new_entries.html'
+    redirect_correct_path = False
 
-    def get_context_data(self, slug, pk):
+    def get_context_data(self, object):
```

Comment:
should you call parent (super) class method first? (Do you want any of parent's context values?)